src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -1051,7 +1051,7 @@ public Metadata createMetadata(String typeName, Metadata metadata) {
      * @return the metadata returned from the server.
      */
     public Metadata createMetadata(String typeName, String scope, Metadata metadata) {
-        URL url = METADATA_URL_TEMPLATE.build(this.getAPI().getBaseURL(), this.getID(), scope, typeName);
+        URL url = METADATA_URL_TEMPLATE.buildAlpha(this.getAPI().getBaseURL(), this.getID(), scope, typeName);
```

Comment:
If `this.getID()` returns, say, "12345abc", then this call would NOT return "An invalid path parameter passed in. It must be numeric." error message, even though that would be an invalid ID.

I can't think of any alternatives, but just calling out to start the discussion.

The API would handle the error in that case, so it's probably a minor concern, right?

Reply:
Ya, you are correct. There are ways around it like passing in what each parameter should be with each parameter in an Array for something like that. However, I think it's a minor concern and isn't worth making the SDK more complicated.